### DISCUSSION TITLE
是否可以下载无字幕的视频？

### DISCUSSION THREAD

**Discussion by curlfoxs** [the1812/Bilibili-Evolved#2811] [功能建议 / Ideas]
请问B站视频的字幕都是内嵌的吗，能否下载无字幕的视频？或者说拿到无字幕视频的URL？
这对我的帮助很大，因为这样就可以添加想要的(双语)字幕，或是在观看视频过程中隐藏字幕用于语言学习。

主要是看了这个视频后感触很大：[MPV - 最好的语言学习播放器[ANKI]](https://www.bilibili.com/video/BV1v64y1D7UZ?from=search&seid=12861412819316006905)
B站有庞大的外语视频资源（英文电影、日文番剧等），MPV播放的问题又基本可得到解决，如果能够下载无字幕的视频，一切都会变的非常诱人。

**Comment by the1812** [comment]
那种支持 CC 字幕的, 视频源才是无字幕的吧

**Reply by curlfoxs** [reply]
好像是的，不过很少，基本是由用户上传视频添加的。

**Reply by the1812** [reply]
就那种直接下下来就是无字幕的, 没有什么新功能要加
